plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -1095,46 +1219,403 @@ function OPDSBrowser:showDownloadListItemDialog(item)
     return true
 end
 
+-- Download whole download or pending_syncs list
 function OPDSBrowser:downloadDownloadList()
-    local info = InfoMessage:new{ text = _("Downloading… (tap to cancel)") }
-    UIManager:show(info)
-    UIManager:forceRePaint()
-    local Trapper = require("ui/trapper")
-    local completed, downloaded = Trapper:dismissableRunInSubprocess(function()
-        local dl = {}
-        for _, item in ipairs(self.downloads) do
-            if self:downloadFile(item.file, item.url, item.username, item.password) then
-                dl[item.file] = true
+    local dl_list
```

Comment:
I'm looking at the `downloadDownloadList()` and it seems that downloads and pending_syncs are quite different.
Maybe leave downloads as is, and make a new method for pending_syncs?